pkg/controller/migmigration/backup.go

```diff
@@ -63,6 +67,55 @@ func (t *Task) ensureBackup() error {
 	return nil
 }
 
+// Ensure the second backup on the source cluster has been created and has the
+// proper settings.
+func (t *Task) ensureCopyBackup() error {
+	newBackup, err := t.buildBackup(nil)
+	if err != nil {
+		return err
+	}
+	uniqueBackupLabelKey := fmt.Sprintf("%s-%s", pvBackupLabelKey, t.PlanResources.MigPlan.UID)
+	labelSelector := metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			uniqueBackupLabelKey:                         "true",
+			fmt.Sprintf("%s-copy", uniqueBackupLabelKey): "true",
+		},
+	}
+
+	// Set included resources to stage resources
+	newBackup.Spec.IncludedResources = stagingResources
+	newBackup.Spec.LabelSelector = &labelSelector
+	foundBackup, err := t.getBackup(true)
+	if err != nil {
+		return err
+	}
+	if foundBackup == nil {
+		t.CopyBackup = newBackup
```

Comment:
This is related to the nomenclature question above around `ensureCopyBackup` -- whatever gets decided there would presumably show up here, too. `t.CopyBackup` looks too much like an action rather than an attribute here.

Reply:
Yeah good point. Can definitely change it to whatever we agree on.

Reply:
We agreed on the phone to instead call this `StageBackup`